templates/sso-cd-image-stream.json

```diff
@@ -47,12 +47,12 @@
                     {
                         "name": "1.0",
                         "annotations": {
-                            "description": "Red Hat Single Sign-On 7.3 continuous delivery Tech Preview image",
+                            "description": "Red Hat Single Sign-On 7.3 Continuous Delivery Tech Preview image",
```

Comment:
Assuming the `7.3` version is kept here just due to backwards compatibility reasons? If that's the case, I am OK with that (IOW this is just a comment if I got it right, rather than something needing action).

Reply:
So this one is the old entry (`5.0`) and it's there for backwards compatibility reasons. 

Frankly, I'm not sure what the policy should be here. Should we leave the current version and one behind it (just in case a critical bug is found)? Or maybe we should leave only the current one?

//cc @hmlnarik

Reply:
Leave what is there for now. We can reconsider later.